code/style.css

```diff
@@ -0,0 +1,147 @@
+@import url("https://fonts.googleapis.com/css2?family=Jost:ital,wght@0,100..900;1,100..900&display=swap");
+
+* {
+  font-family: "Jost", sans-serif;
+  margin: 0;
+  box-sizing: border-box;
+  background-color: #fafbff;
+}
+
+h1 {
+  color: #0018a4;
+  margin: 20px;
+  padding-left: 40px;
+  font-size: 64px;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+}
+
+.wrapper {
+  display: flex;
+  justify-content: flex-start;
+}
+
+.filter-box,
+.sort-box,
+.random-button-box {
+  margin: 2rem;
+  padding-left: 30px;
+}
+
+.headings-top {
+  font-size: 22px;
+  font-weight: bold;
+  padding: 16px 16px 16px 0px;
+}
+
+.filter-buttons:focus,
+#random:focus {
+  background-color: #0018a4;
+  color: #ffffff;
+}
+
+.sort-buttons:focus {
+  background-color: #ff6589;
+  color: #ffffff;
+}
+
+.filter-buttons,
+#random {
+  background-color: #ccffe2;
+  color: #0018a4;
+  padding: 8px 16px 8px 16px;
+  font-style: bold;
+  border-radius: 50px;
+  border: none;
+}
+
+.filter-buttons:hover,
+#random:hover {
+  border: 2px solid #0018a4;
+}
```

Comment:
same here. Hover effects should not affect the positioning of the rest of the UI